carma-messenger-core/cpp_message/CMakeLists.txt

```diff
@@ -60,9 +60,11 @@ add_executable(cpp_message_node
 target_link_libraries(cpp_message_node testlib ${Boost_LIBRARIES} ${catkin_LIBRARIES} cpp_message_library)
 
 add_library(cpp_message_library src/cpp_message.cpp 
-            ##src/MessageConsumer.cpp
+			##src/MessageConsumer.cpp
```

Comment:
Remove commented out lines that will not be used

Reply:
Updated